docs/getting-started/connect_to_cloud.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Connect to Soda Cloud
+parent: Getting Started
+nav_order: 3
+---
+
+# Connect to Soda Cloud
+
+To use the Soda Cloud web user interface to monitor your data, you must install and configure the soda-sql command line tool, then connect it to your Soda Cloud account.
+
+* The **soda-sql** command line tool does the work of scanning the data in your database. 
+* Your **Soda Cloud** account is where soda-sql pushes the results of its scans. Use the web UI to examine the results and set up alerts.  
+
+soda-sql uses an API to connect to Soda Cloud. To use the API, you must generate API keys in your Soda Cloud account, then add them to the `warehouse.yml` file that soda-sql created. 
+
+
+
+1. If you have not already done so, create a Soda Cloud account at `cloud.soda.io`.
```

Comment:
You might want to link `cloud.soda.io` directly to the signup page (`https://cloud.soda.io/signup` ), since the login page doesn't contain a 'signup' link at the moment. :eyes:

Reply:
Ah, OK, makes sense. Will do.